src/sparseml/transformers/finetune/example_alternating_recipe.yaml

```diff
@@ -0,0 +1,54 @@
+initial_sparsity_stage:
+  run_type: oneshot
+  obcq_modifiers:
+    SparseGPTModifier:
+      sparsity: 0.5
+      block_size: 128
+      sequential_update: False
+      quantize: False
+      percdamp: 0.01
+      prunen: 0
+      prunem: 0
+      targets: [
+        "re:model.layers.\\d+$"
+      ]
+initial_training_stage:
+  run_type: train
+  pruning_modifiers:
+    ConstantPruningModifier:
+      targets: [
+        "re:.*self_attn.q_proj",
+        "re:.*self_attn.k_proj",
+        "re:.*self_attn.v_proj",
+        "re:.*self_attn.o_proj",
+        "re:.*mlp.gate_proj",
+        "re:.*mlp.up_proj"
+      ]
+      start: 0
+next_sparsity_stage:
+  run_type: oneshot
+  obcq_modifiers:
+    SparseGPTModifier:
+      sparsity: 0.7
+      block_size: 128
+      sequential_update: False
+      quantize: False
+      percdamp: 0.01
+      prunen: 0
+      prunem: 0
+      targets: [
+        "re:model.layers.\\d+$"
+      ]
+next_training_stage:
+  run_type: train
+  pruning_modifiers:
+    ConstantPruningModifier:
+      targets: [
+        "re:.*self_attn.q_proj",
+        "re:.*self_attn.k_proj",
+        "re:.*self_attn.v_proj",
+        "re:.*self_attn.o_proj",
+        "re:.*mlp.gate_proj",
+        "re:.*mlp.up_proj"
+      ]
```

Comment:
why can't we just use `_ALL_` here like we used previous for sparse transfer?

Reply:
Yeah good point, `__ALL__` should work but let me test it quickly first